platform/linux-dpdk/include/odp/api/plat/version_types.h

```diff
@@ -0,0 +1 @@
+../../../../../linux-generic/include/odp/api/plat/version_types.h
```

Comment:
This way we are doing two merges to work with cloud environment,
one merge from api-next to cloud-dev and then cloud-dev to odp-dpdk-import. This seems more work and confusing as well in a way that same group working with cloud will work in two different branches or places.